app/policies/mission_policy.rb

```diff
@@ -0,0 +1,12 @@
+class MissionPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+
+  def index?
```

Comment:
@Nicode80 ,I don't think need to enable index this way (it uses authorize). If go to `app/controller/applications_controller.rb` you'll see that auhtorize is not set for index. Index only uses scope policy.